config/dca/cadesGPUCondo.cmake

```diff
@@ -1,5 +1,4 @@
-#  Copyright (c) 2018      Peter Doak
-#  Copyright (c) 2017-2018 John Biddiscombe
+#  Copyright (c) 2018 John Biddiscombe, Peter Doak
```

Comment:
trivial detail, but I quite like the hpx convention where each contributor puts his name on a separate line and edits the dates accordingly.

Reply:
Yest that's more informative, I'll switch it back

Reply:
So last person to update file on top?